modules/aws/centos-gfx/centos-gfx-startup.sh.tmpl

```diff
@@ -251,6 +237,22 @@ join_domain()
         sed -i '$ a\dyndns_update = True\ndyndns_ttl = 3600\ndyndns_refresh_interval = 43200\ndyndns_update_ptr = True\nldap_user_principal = nosuchattribute' /etc/sssd/sssd.conf
         sed -c -i "s/\\(use_fully_qualified_names *= *\\).*/\\1False/" /etc/sssd/sssd.conf
         sed -c -i "s/\\(fallback_homedir *= *\\).*/\\1\\/home\\/%u/" /etc/sssd/sssd.conf
+
+        # sssd.conf configuration is required first before enabling sssd
```

Comment:
this comment could be confusing when the reader didn't know we moved it from before joining domain.